_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -50089,8 +50066,8 @@
 	},
 /obj/item/gun/energy/laser,
 /obj/item/gun/energy/laser{
-	pixel_x = 3;
-	pixel_y = -3
+	pixel_x = 0;
+	pixel_y = 0
```

Comment:
This increases memory usage and floods the travis output logs. You can safely remove it